build/config/compiler/BUILD.gn

```diff
@@ -668,6 +618,20 @@ config("runtime_library") {
       libs += [ "atomic" ]
     }
   }
+
+  # Linux standard liburary setup.
+  if (is_linux) {
+    cflags_cc += [
+      "-stdlib=libc++"
+    ]
+    ldflags += [
+      "-stdlib=libc++",
+      "-static-libstdc++",
+      "-Wl,-Bstatic", "-lc++abi",
```

Comment:
I expect we'll be able to remove some or all of these lines soon, assuming Petr cleans up our toolchain distro. Can you leave a TODO or some other comment here reminding the next buildtools roller to reconsider these?